README.md

```diff
@@ -40,8 +40,9 @@ Be sure to remove or modify the `username` attribute.
 
 ```sh
 $ bundle exec rake db:create \
->  && bundle exec rake db:schema:load \
->  && bundle exec rake db:seed
+  && bundle exec rake db:migrate \
```

Comment:
Yes you are right of course. However, there are some that cannot be stacked in the same call. It's long enough ago that I can't remember the exact issue, I only remember that `rake db:drop db:create && rake db:migrate` must be done in separate invocations otherwise migrations from engines don't get applied. Have you seen this too? Shall we work out a safe split or shall we leave it? @benlovell